src/decision_process/examples.md

```diff
@@ -0,0 +1,297 @@
+# Examples of the decision-making process in action
+
+## Reversible decision: merging a proposal
+
+The process is best described by example. Suppose that there is a pending lang
+team proposal, and a lang team member would like to serve as the liaison. They
+contact the team leads and receive the go-ahead. They can then write:
+
+> @rustbot merge
+>
+> I propose to merge this proposal. I think it will be a great addition to
+> Rust!
+
+This indicates that they would like to merge the proposal. At the moment, there
+is no decision pending, so rustbot would add a comment that looks like the
+following:
+
+> Hello! @Alan has proposed to merge this. This is a **reversible decision**,
+> which means that it will be affirmed once the "final comment period" of 10
+> days have passed, unless a team member places a "hold" on the decision (or
+> cancels it).
+>
+> | Team member | State |
+> | --- | --- |
+> | @Alan | **merge** |
+> | @Barbara | |
+> | @Grace | |
+> | @Niklaus | |
+
+As the comment says, the PR is now in "pending decision" state, with Alan
+having kicked off the process with a proposal to merge. Alan's status of
+**merge** will link to his comment.
+
+Now, for this particular proposal, Barbara has a concern. She thinks that the
+proposal has overlooked an important consideration. She writes a comment:
+
+> @rustbot hold
+>
+> Did you consider reversing the polarity? Or the impact on the flux capacitor?
+
+At this point, rustbot updates the state:
+
+> Hello! @Alan has proposed to merge this. This is a **reversible decision**,
+> which means that it will be affirmed once the "final comment period" of 10
+> days have passed, unless a team member pauses or suspends the decision.
```

Comment:
I cannot tell if you have deliberately changed the rustbot authored text from what was written above, or if the text here is meant to be kept in sync with the text above. I assume the latter, and will write a suggestion accordingly.

Reply:
```suggestion
> days have passed, unless a team member places a "hold" on the decision (or cancels it).
```

Reply:
The reason I took the time to correct this in every occurrence I could find (see below) is that I think it is important that the rustbot text uses the same terminology that appears in the table and/or in our commands.

I.e. it is confusing for rustbot to talk about "pauses or suspends" and then for a third word, "hold", to show up in the table.